### DISCUSSION TITLE
关于高考填报志愿大类招生的一些想法

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#26] [Ideas]
以下内容转自群友

---

> 务必注意大类招生这个坑，以免误入不喜欢的专业。有些学校为了避免被冷门专业拉低分数线，采用大类招生方式，也就是把多个专业绑定成一个大类招生。高考填报志愿只能决定你进入哪个大类，不保证你进入特定的专业。待上大学后，根据大一期末考试或者平时的成绩再分流专业。
>
> 拿中山大学来说，材料、土木和计算机专业在同一个大类“计算机类”里招生。想学计算机，高考考进中山大学“计算机类”大类的考生，如果上大学后的成绩不好，最终会被分流进土木、材料专业。分流考试的竞争会比高考更加激烈，毕竟高考有很多水平比我低的考生，但大学里都是和我高考分数差不多的人。
>
> 如果我目标范围内很多学校都采取大类招生的话，那么我就需要从现在开始预习并备考大一上、下学期的期末考试了，像高考一样去认真准备。因为决定专业这件事从高考转移到了大一期末考试。往年试卷及大一的课程在网上可以找到。

以上是个人在准备高考填报志愿期间的一些经验和想法，其中的技巧是我近两天才速成掌握的，不一定准确。我毕竟也还没上大学，只是依靠网络尽量去了解了一下大学的各方面。以下内容仅为个人日记性质，并非保证准确可靠的教程。由于我已确定了报考计算机科学与技术专业或与之近似的专业，因此填报志愿围绕我的个人目标开展。

**Comment by 794TD** [comment]
非985/211的一些报考建议
- 如果报不上985或者211那就要在剩下的学校里挑专业排名好的，可以参考全国学科等级评估。
学校是不是双非决定的是HR会不会看你的简历，真正比拼的是个人能力。
- 不推荐为了985和211的名头去报诸如新疆石河子大学等。
- 如果两所学校排名相近的话，能去大城市就去大城市，大城市里面学校的机会（比赛、实习）会多一些。
- 冲双一流的时候一定要分清，是**学科是双一流学科还是学校是双一流学校**。
- 双非不代表认可度低，有些双非的某些专业属于行业内的清华北大，如果想要从事相关行业，这其实也是不错的选择（比如大连海事学院、天津的民航大和广汉的飞行院都是在各自行业内的重要学府）。
- 想要深造读研的话记得关注保研之类的事宜。

**Reply by KokuraSona** [reply]
你好，我打算报考计算机相关专业，对于你说的第二点，想问一下“诸如新疆石河子大学”是指末流211吗？以及我同样关注了两所末流211（东北林业大学、东北农业大学）想知道这两所大学是否也属于你所说的范畴内？希望能得到一些建议，感激不尽。

**Reply by 794TD** [reply]
可以仔细查一下这俩学校计算机专业的学科评估报告，如果计算机有B-以上评级我觉得就可以。我查到的是东北林业大学的计算机没有参加学科评估（也就是说有可能摸不到平均线）。
如果深造考研的话，关注一下考研的信息，有人说211考研会容易，但是有些双非学校保研也容易。
`学校是不是双非决定的是HR会不会看你的简历，真正比拼的是个人能力。`
如果本科毕业就工作，那推荐去专业水平好的学校，如果为了211考研的名头，那考虑一下这俩也可以。

**Reply by 794TD** [reply]
新疆石河子大学对于大部分人来说绝对不是最优解，不是因为水平，是因为地区问题，太偏了。就比如计算机大四实习，新疆那边肯定不如东部地区

**Reply by KokuraSona** [reply]
非常感谢🙏

**Reply by Puke-il** [reply]
您好，那青海大学算是偏远211吗？
